integration-libs/opf/components/opf-checkout-billing-address-form/opf-checkout-billing-address-form.component.html

```diff
@@ -1,67 +1,61 @@
 <ng-container *ngIf="!(isLoadingAddress$ | async); else loading">
-  <div class="form-group">
-    <div class="form-check">
-      <label>
-        <input
-          type="checkbox"
-          class="form-check-input"
-          [checked]="isSameAsDelivery"
-          (change)="toggleSameAsDeliveryAddress()"
-          [attr.aria-label]="
-            'paymentForm.billingAddressSameAsShipping' | cxTranslate
-          "
-        />
-        <span class="form-check-label">{{
-          'paymentForm.sameAsDeliveryAddress' | cxTranslate
-        }}</span>
-      </label>
-    </div>
-  </div>
-
   <ng-container
-    *ngIf="
-      !isEditBillingAddress &&
-      isSameAsDelivery &&
-      (deliveryAddress$ | async) as deliveryAddress
-    "
+    *ngIf="{
+      billingAddress: billingAddress$ | async,
+      isSameAsDelivery: isSameAsDelivery$ | async
+    } as observables"
```

Comment:
Naming this `as observables` does not sound good to me, because we don't have an access to observables but to their values.
What about a change to `data` or `addressData`?

Reply:
Thanks, changed